src/app/components/grid/index.html

```diff
@@ -103,29 +160,281 @@
+		</stache-column>
+		<stache-column screenSmall="6">
+			<div class="screenshot screenshot-info">
+				<img src="~/assets/img/guidelines/grid/grid-option-template-column.png" style="width: 100%;" />
+			</div>
+		</stache-column>
+	</stache-row>
+    
+	<header><div class="sky-section-heading">Behavior</div></header>
+
+	<header><div class="sky-subsection-heading">Fit</div></header>
+	
+	<header><div class="sky-emphasized">Scroll</div></header>	
+		
+	
+		<stache-row>
+			<stache-column screenSmall="6">
+				<p>Setting the <stache-code>fit</stache-code> property to <stache-code>scroll</stache-code> allows the grid to take as much space as necessary by adding a horizontal scrollbar. This is most useful for grids with large numbers of columns.</p>
+			</stache-column>
+			<stache-column screenSmall="6">
+			<div class="screenshot screenshot-info">
+				<img src="~/assets/img/guidelines/grid/grid-fit-scroll.png" style="width: 100%;" />
+			</div>
+		</stache-column>
+		</stache-row>	
+		
+	
+	<header><div class="sky-emphasized">Width</div></header>
+  
+	
+		<stache-row>
+			<stache-column screenSmall="6">
+				<p>Setting the <stache-code>fit</stache-code> property to <stache-code>width</stache-code> fixes the width of the grid to that of its container. Columns resize themselves to fit. This is most useful for simple grids in smaller containers such as tiles where the content fits neatly in the allotted space.</p>
+			</stache-column>
+			<stache-column screenSmall="6">
+			<div class="screenshot screenshot-info">
+				<img src="~/assets/img/guidelines/grid/grid-fit-width.png" style="width: 100%;" />
+			</div>
+		</stache-column>
+		</stache-row>	
+		
+	
+	<header><div class="sky-subsection-heading">Reordering columns</div></header>
+
+  <stache-row>
+    <stache-column screenSmall="6">
+			<p>Users can reorder columns by dragging and dropping.</p>
+		</stache-column>
+		<stache-column screenSmall="6">
+			<div class="screenshot screenshot-success">
+				<img src="~/assets/img/guidelines/grid/grid-reorder.png" style="width: 100%;" />
+			</div>
+		</stache-column>
+	</stache-row>
+    
+	<header><div class="sky-subsection-heading">Resizing columns</div></header>
+
+  <stache-row>
+    <stache-column screenSmall="6">
+			<p>Users can resize columns by clicking and dragging or by using the keyboard.</p>
+		</stache-column>
+		<stache-column screenSmall="6">
+			<div class="screenshot screenshot-success">
+				<img src="~/assets/img/guidelines/grid/grid-resize.png" style="width: 100%;" />
+			</div>
+		</stache-column>
+	</stache-row>
+    
+	<header><div class="sky-section-heading">Responsiveness</div></header>
+  <p>Grids tend to perform poorly in small viewports. The columns in fixed width grids will shrink down to the point of being unreadable. Scrolling grids will require a lot of interaction to see all the content.</p>
```

Comment:
This paragraph is nice, but I'm left wondering what is the alternative? Is there a solution or suggested path to take here if I want to display data in small viewports?

This might not be a question we have to answer today, but thought it was worth mentioning that this is the last thing on the page, and it kind of leaves me hanging :)

Reply:
added a link to use repeater instead.